src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -1410,6 +1424,17 @@ def getPrepID(self):
         """
         return getattr(self.data, 'prep_id', "")
 
+    def getWMTiming(self):
+        """
+         _getWMTiming_
```

Comment:
Please refresh this contribution guidelines: https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices

It seems you also have trailing spaces here.

Reply:
will do, I copied docstring as it is used in other parts in this code. And, new docstring style will differ from other methods just saying.

Reply:
That's fine. At least we will be slowly converging towards one style.

Reply:
ping